base/db/R/query.prior.R

```diff
@@ -22,6 +22,7 @@
 ##' @param con Database connection object.
 ##' @param ... Optional arguments for connecting to database (e.g.
 ##'   password, user name, database).
+##' @param update 
```

Comment:
The current CI failure is from Roxygen complaining that this parameter description is empty